install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,627 @@ $ oc label node --all logging-infra-fluentd=true <1>
+
+| `systemd.t.SYSLOG_PID`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#SYSLOG_FACILITY=[`SYSLOG_PID`]
+is the client process ID for `syslog`.
+|===
+
+[discrete]
+[[exported-fields-systemd.u]]
+=== `systemd.u` Fields
+
+`systemd.u Fields` are directly passed from clients and stored in the journal.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `systemd.u.CODE_FILE`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#CODE_FILE=[`systemd.u.CODE_FILE`]
+is the code location containing the filename of the source.
+
+| `systemd.u.CODE_FUNCTION`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#CODE_FILE=[`systemd.u.CODE_FUNCTION`]
+is the code location containing the function of the source.
+
+| `systemd.u.CODE_LINE`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#CODE_FILE=[`systemd.u.CODE_LINE`]
+is the code location containing the line number of the source.
+
+| `systemd.u.ERRNO`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#ERRNO=[`systemd.u.ERRNO`],
+if present, is the low-level error number formatted in numeric value, as a decimal string.
+
+| `systemd.u.MESSAGE_ID`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#MESSAGE_ID=[`systemd.u.MESSAGE_ID`]
+is the message identifier ID for recognizing message types.
+
+| `systemd.u.RESULT`
+|For private use only.
+
+| `systemd.u.UNIT`
+|For private use only.
+|===
+
+[discrete]
+[[exported-fields-kubernetes]]
+=== Kubernetes Fields
+
+The namespace for Kubernetes-specific metadata. The `kubernetes.pod_name` is the
+name of the pod.
+
+[discrete]
+[[exported-fields-kubernetes.labels]]
+=== `kubernetes.labels` Fields
+
+Labels attached to the OpenShift object are `kubernetes.labels`. Each label name
+is a subfield of labels field. Each label name is de-dotted, meaning dots in the
+name are replaced with underscores.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `kubernetes.pod_id`
+|Kubernetes ID of the pod.
+
+| `kubernetes.namespace_name`
+|The name of the namespace in Kubernetes.
+
+| `kubernetes.namespace_id`
+|ID of the namespace in Kubernetes.
+
+| `kubernetes.host`
+|Kubernetes node name.
+
+| `kubernetes.container_name`
+|The name of the container in Kubernetes.
+
+| `kubernetes.labels.deployment`
+||The deployment associated with the Kubernetes object.
+
+| `kubernetes.labels.deploymentconfig`
+|The deploymentconfig associated with the Kubernetes object.
+
+| `kubernetes.labels.component`
+|The component associated with the Kubernetes object.
+
+| `kubernetes.labels.provider`
+|The provider associated with the Kubernetes object.
+|===
+
+[discrete]
+[[exported-fields-kubernetes.annotations]]
+=== `kubernetes.annotations` Fields
+
+Annotations associated with the OpenShift object are `kubernetes.annotations`
+fields.
+
```

Comment:
I thought so too, but it looks like it's on it's own: https://github.com/ViaQ/elasticsearch-templates/blob/07403406b4077f90be6f24ec8234bfaea2f83ba3/namespaces/kubernetes.yml